.circleci/config.yml

```diff
@@ -109,14 +109,16 @@ workflows:
       - aws_destroy_by_name
 
   nightly:
-    triggers:
-      - schedule:
-          cron: "0 5 * * *"
-          filters:
-            branches:
-              only:
-                - "2.0"
+    #triggers:
```

Comment:
Reminder to change this testing code back prior to merging

Reply:
:white_check_mark: